tests/test_cmdline.py

```diff
@@ -21,6 +21,27 @@
 logger = logging.getLogger(__name__)
 
 
+def make_settings_dict(**kwargs):
+    """Create an expected version of Settings.dict(), with customizations.
+
+    Return a copy of DEFAULTS, with the given customizations applied.
+    """
+    default_values = {
+        "actions": ["check_undeclared", "check_unused"],
+        "code": ["."],
+        "deps": ["."],
+        "pyenv": None,
+        "output_format": "human_summary",
+        "ignore_undeclared": [],
+        "ignore_unused": [],
+        "deps_parser_choice": None,
+        "verbosity": 0,
+    }
+    ret = default_values.copy()
+    ret.update(kwargs)
+    return ret
```

Comment:
The extra `.copy()` is not needed here when the dict is local to this function (and thus a new dict is create on each call):

```suggestion
    ret = {
        "actions": ["check_undeclared", "check_unused"],
        "code": ["."],
        "deps": ["."],
        "pyenv": None,
        "output_format": "human_summary",
        "ignore_undeclared": [],
        "ignore_unused": [],
        "deps_parser_choice": None,
        "verbosity": 0,
    }
    ret.update(kwargs)
    return ret
```

Reply:
Done in [b4398da](https://github.com/tweag/FawltyDeps/pull/261/commits/b4398da4fa25c129c7fd37b78c1f01a6e43b0340).